### PR TITLE
Add documentation for the SPDE mini app

### PR DESCRIPTION
**Pull request by TobiasDuswald**
This PR is a companion to https://github.com/mfem/mfem/pull/3065

### CODE REVIEW

**Comment by TobiasDuswald** [comment]
@brendankeith @bslazarov Unfortunately, I was not able to set up the `mkdocs` / `python` environment to check that everything renders alright. If you have it setup on your computer, could you check that?

File: src/features.md [hunk 1]
```diff
@@ -140,6 +140,8 @@ Beyond the examples, a number of miniapps are available that are more representa
  - [Convert DC](https://docs.mfem.org/html/convert-dc_8cpp_source.html): convert between different DataCollection formats,
  - [LOR Transfer](https://docs.mfem.org/html/lor-transfer_8cpp_source.html): map functions between high-order and low-order-refined spaces,
  - [ParELAG H(curl) and H(div) AMGe](https://docs.mfem.org/html/MultilevelHcurlHdivSolver_8cpp_source.html): solve H(curl) and H(div) problems using the element-based algebraic multigrid (AMGe) in the ParELAG library.
+ - [SPDE](ToDo:AddLink):
```
Comment (brendankeith) [inline src/features.md:143]: Requires a link

**Review by brendankeith — APPROVED** [review]
This looks great, @TobiasDuswald!

**Comment by tzanio** [comment]
Thanks guys!

**Review by tzanio — APPROVED** [review]

**Pull request merged**